application-provider/src/test/java/org/jboss/intersmash/applications/ApplicationProviderTest.java

```diff
@@ -154,6 +170,21 @@ void wildflyDistributedSessionsInfinispanDeployment_exists() {
 		assertPathExist(appPath);
 	}
 
+	/**
+	 * Verify that a WAR deployment for the
+	 * {@code wildfly-amq-broker-ssl} application exists.
+	 */
+	@Test
+	void wildflyActiveMQArtemisDeploymentPath_exists() {
```

Comment:
Rename `wildflyActiveMQArtemisDeploymentPath_exists` to `wildflyActiveMQArtemisSslDeploymentPath_exists`?